examples/tutorial/01_Overview.ipynb

```diff
@@ -151,11 +151,11 @@
    "source": [
     "Each of the HoloViz tools provides its own API for accessing whatever functionality it implements (and sometimes more than one API, for different purposes!). For this tutorial, we will focus on the following APIs that we believe provide the most power to new users for the smallest investment of time and effort:\n",
     "\n",
-    "* [hvPlot](https://hvplot.pyviz.org) `.plot()` API, which adds additional power to a well-known API supported by many different data libraries. \n",
-    "* [hvPlot](https://hvplot.pyviz.org) `.interactive()` API, which brings live interactivity to the existing Pandas and Xarray APIs.\n",
+    "* [hvPlot](https://hvplot.holoviz.org) `.plot()` API, which adds additional power to a well-known API supported by many different data libraries. \n",
+    "* [Param](https://param.holoviz.org) `.rx()` API, which brings live interactivity to the existing Pandas and Xarray APIs.\n",
```

Comment:
Not sure this is how Jim would define `rx()`, he'll anyway be able to update this when he'll review the tutorial.